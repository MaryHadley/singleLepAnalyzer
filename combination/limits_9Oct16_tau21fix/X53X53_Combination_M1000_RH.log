000 (1295370.000.000) 10/09 22:41:05 Job submitted from host: <192.168.63.249:41533>
...
001 (1295370.000.000) 10/09 22:41:26 Job executing on host: <192.168.63.231:41752>
...
006 (1295370.000.000) 10/09 22:41:34 Image size of job updated: 1
	3  -  MemoryUsage of job (MB)
	2272  -  ResidentSetSize of job (KB)
...
006 (1295370.000.000) 10/09 22:46:34 Image size of job updated: 283576
	145  -  MemoryUsage of job (MB)
	147704  -  ResidentSetSize of job (KB)
...
006 (1295370.000.000) 10/10 00:01:42 Image size of job updated: 283576
	145  -  MemoryUsage of job (MB)
	147716  -  ResidentSetSize of job (KB)
...
006 (1295370.000.000) 10/10 17:48:30 Image size of job updated: 297860
	157  -  MemoryUsage of job (MB)
	160680  -  ResidentSetSize of job (KB)
...
005 (1295370.000.000) 10/10 18:02:50 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 19:19:30, Sys 0 00:00:04  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 19:19:30, Sys 0 00:00:04  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	1596  -  Run Bytes Sent By Job
	307  -  Run Bytes Received By Job
	1596  -  Total Bytes Sent By Job
	307  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :    31774        1   3824812
	   Memory (MB)          :      157     3072      3072
...
