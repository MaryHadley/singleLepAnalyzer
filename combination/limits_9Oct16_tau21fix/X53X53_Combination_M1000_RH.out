/var/lib/condor/execute/dir_21767
Total number of likelihood evaluations: 1100000101
Total number of likelihood evaluations: 11000101
[INFO]  executing script /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1000_RH.py
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1000_RH/quant-toys:0-sig-aec6706d8b.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1000_RH/quant-data-sig-cb952cabdc.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  workdir is /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1000_RH
