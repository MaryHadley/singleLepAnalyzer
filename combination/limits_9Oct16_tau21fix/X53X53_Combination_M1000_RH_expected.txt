# x; y; band 0 low; band 0 high; band 1 low; band 1 high
         0   0.073536   0.036899    0.15381  0.050782    0.10856
